xml/ceph_maintenance_updates.xml

```diff
@@ -33,6 +33,125 @@
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    CVE-2020-1760: Fixed XSS due to RGW GetObject header-splitting
+   </para>
+  </listitem>
+ </itemizedlist>
+ <para>
+  In SES 6, these flaws were patched in &ceph; version 14.2.5.389+gb0f23ac248.
+ </para>
+ <bridgehead renderas="sect1">&cephname; 14.2.8 Point Release</bridgehead>
+ <para>
+  In addition to bug fixes, this major upstream release brought a number of
+  notable changes:
+ </para>
+ <itemizedlist>
+  <listitem>
+   <para>
+    The default value of <option>bluestore_min_alloc_size_ssd</option> has been
+    changed to 4K to improve performance across all workloads.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    The following OSD memory config options related to &bluestore; cache
+    autotuning can now be configured during runtime:
+   </para>
+<screen>
+osd_memory_base (default: 768 MB)
+osd_memory_cache_min (default: 128 MB)
+osd_memory_expected_fragmentation (default: 0.15)
+osd_memory_target (default: 4 GB)
+</screen>
+   <para>
+    You can set the above options by running:
+   </para>
+<screen>&prompt.cephuser;ceph config set osd <replaceable>OPTION</replaceable> <replaceable>VALUE</replaceable></screen>
+  </listitem>
+  <listitem>
+   <para>
+    The &mgr; now accepts <literal>profile rbd</literal> and <literal>profile
+    rbd-read-only</literal> user capabilities. You can use these capabilities
+    to provide users access to MGR-based RBD functionality such as <literal>rbd
+    perf image iostat</literal> and <literal>rbd perf image iotop</literal>.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    The configuration value <option>osd_calc_pg_upmaps_max_stddev</option> used
+    for upmap balancing has been removed. Instead, use the &mgr; balancer
+    configuration option <option>upmap_max_deviation</option> which now is an
+    integer number of PGs of deviation from the target PGs per OSD. You can set
+    it with a following command:
+   </para>
+<screen>&prompt.cephuser;ceph config set mgr mgr/balancer/upmap_max_deviation 2</screen>
+   <para>
+    The default <option>upmap_max_deviation</option> is 5. There are situations
+    where crush rules would not allow a pool to ever have completely balanced
+    PGs. For example, if crush requires 1 replica on each of 3 racks, but there
+    are fewer OSDs in 1 of the racks. In those cases, the configuration value
+    can be increased.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    &cephfs;: multiple active &mds; forward scrub is now rejected. Scrub
+    currently only is permitted on a file system with a single rank. Reduce the
+    ranks to one via <command>ceph fs set <replaceable>FS_NAME</replaceable>
+    max_mds 1</command>.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    &ceph; will now issue a health warning if a &rados; pool has a
+    <option>pg_num</option> value that is not a power of two. This can be fixed
+    by adjusting the pool to a nearby power of two:
```

Comment:
```suggestion
    by adjusting the pool to an adjacent power of two:
```

Reply:
I am not sure about this; it is a guess.

"Nearby" could mean 1-2 away, which could be a very big step! "Adjacent" implies the next nearest, i.e. rounding up or rounding down, which would seem to make more sense to me.

Reply:
The upstream release notes say "nearby", but I think "adjacent" is OK, too.